src/Sylius/Bundle/ApiBundle/DataProvider/Helpers/ProductDataProviderHelper.php

```diff
@@ -18,14 +19,18 @@ public static function setCustomPropertiesToProduct($product, $channelCode)
         $prices = [];
 
         foreach ($product->getVariants()->getValues() as $i => $variant) {
+
+            if (!is_object($variant->getChannelPricings()[$channelCode])) {
+                break;
+            }
             $prices[] = $variant->getChannelPricings()[$channelCode]->getPrice();
             $onHand += $variant->getOnHand();
         }
```

Comment:
Are you sure that `break` is correct condition operator? Maybe `continue` will be better than `break`?

```suggestion
        foreach ($product->getVariants()->getValues() as $i => $variant) {

            if (!is_object($variant->getChannelPricings()[$channelCode])) {
                continue;
            }
            $prices[] = $variant->getChannelPricings()[$channelCode]->getPrice();
            $onHand += $variant->getOnHand();
        }
```
Or describe me please why `break`?

Reply:
GJ